### PR TITLE
Add comment to card

### PR DESCRIPTION
**Pull request by gusehr**
@julienvey Hi there.
Thanks for the trello-java-wrapper ;)

I just added an "add comment" feature to a Trello card.
Can you add it to the trello-java-wrapper?

Thanks in advance.

### CODE REVIEW

File: src/main/java/com/julienvey/trello/Trello.java [hunk 1]
```diff
@@ -89,6 +89,8 @@
 
     void addLabelsToCard(String idCard, String[] labels);
 
+	void addCommentToCard(String idCard, String comment);
```
Comment (julienvey) [inline src/main/java/com/julienvey/trello/Trello.java]: Minor fix, but this line seems to have a bad indentation

Comment (gusehr) [inline src/main/java/com/julienvey/trello/Trello.java]: OK! @julienvey 
Fixed

**Comment by julienvey** [comment]
Thanks @gusehr 

I will release a new minor version soon, so you can use directly a released version

**Comment by gusehr** [comment]
Great @julienvey !
I'm working on post new attachments too, but still in progress.

**Pull request merged**